tip-wlan/example-values/local-multi-namespace/README.md

```diff
@@ -3,8 +3,8 @@
 ## Usage
 
 ```bash
-helm install tip-wlan-1 tip-wlan -f tip-wlan/example-values/local-multi-namespace/ns-tip-1.yaml
-helm install tip-wlan-2 tip-wlan -f tip-wlan/example-values/local-multi-namespace/ns-tip-2.yaml
+helm install tip-wlan-1 tip-wlan -f tip-wlan/example-values/local-multi-namespace/ns-tip-1.yaml --namespace tip-wlan-1 --create-namespace
```

Comment:
Maybe we should update the docs to use `helm upgrade --install` instead of the `helm install` command? The upgrade command is idempotent which is better.